api/v1alpha3/condition_types.go

```diff
@@ -95,3 +95,14 @@ type Condition struct {
 type Conditions []Condition
 
 // ANCHOR_END: Conditions
+
+// CreateReplicasUpdatedCondition creates a static ReplicasUpdated Condition
+func CreateReplicasUpdatedCondition(message string) Condition {
```

Comment:
For other conditions we don't have Create function inside the API; instead, we rely on `conditions.MarkTrue` or `conditions MarkFalse` utilities